sample-code/ruby/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://www.rubygems.org'
+
+gem 'appium_lib',         '~> 9.7.4'
```

Comment:
The latest version is `9.10`. Could you set the line as `'~> 9.10'`?
It means we can download the latest `9.x` version.